gcs/CONFIGURATION.md

```diff
@@ -22,7 +22,7 @@
     Whether to perform copy operation using Rewrite requests which allows copy
     files between different locations and storage classes.
 
-*   `fs.gs.rewrite.max.bytes.per.call` (default: `536870912`)
```

Comment:
A general question. What is the issue with what is already present? i.e why fixing something which is not broken.

Reply:
It's quite hard to use this properties - we constantly need to convert gb, mb, etc to bytes. Also, this is a standard functionality for Hadoop properties, so it could be surprising for users that they can not use units in properties.